sympy/calculus/util.py

```diff
@@ -1290,14 +1297,18 @@ def __rtruediv__(self, other):
 
     @_sympifyit('other', NotImplemented)
     def __pow__(self, other):
-        from sympy.functions.elementary.miscellaneous import real_root
         if isinstance(other, Expr):
             if other is S.Infinity:
                 if self.min.is_extended_nonnegative:
                     if self.max < 1:
                         return S.Zero
                     if self.min > 1:
                         return S.Infinity
+                    # XXX is 1**oo to be considered oo or 1?
```

Comment:
The result should be `AccumBounds(0, 00)` also in these cases as there are (real) functions `f` and `g` that have limits `1`resp. `oo` but `f(t)**g(t)` may have any nonnegative real as limit or, more generally, as accumulation value, if a limit does not exist . If `h` is any function whose accumulation values (at the same point) are described by `AccumBounds(0, 2)` (like `sint(t) + 1` at `oo`) then `f(t) + h(t)` has limit `AccumBounds(1, 3)` and `(f(t) + h(t))**g(t)` may have any nonnegative extended real as accumulation value (`f(t) + h(t)` may be < 1 repeatedly).